packages/teleport-test/src/standalone.ts

```diff
@@ -203,145 +202,6 @@ const run = async () => {
       })
       return `React - StyledComponents`
     })
-
```

Comment:
Is this supposed to be removed?

Reply:
Yeah, we are not actively working on tailwind in generators anymore. So, it's just a lot of noise everytime we are testing the project developent.